### PR TITLE
Remove tutorials CMSIS dependency when not needed

### PR DESCRIPTION
**Pull request by NicolaLancellotti**
Pr #13656 added support for CMSIS NN from a new GitHub location.
The project at the new location uses a new header `arm_acle.h` which is not always present.
This pr removes the CMSIS dependency for the tutorials in `gallery/how_to/work_with_microtvm` when the tutorials are run with Zephyr.
We need this pr, otherwise, when the docker image will be updated there will be errors, due to the missing header.

### CODE REVIEW

**Comment by NicolaLancellotti** [comment]
@ashutosh-arm

**Review by guberti — CHANGES_REQUESTED** [review]
This looks good, but do these tutorials still need to include instructions to install CMSIS? I'd love to remove this from `micro_aot.py`, `micro_autotune.py`, and `micro_tflite.py` if we can.

```python
######################################################################
#
#     .. include:: ../../../../gallery/how_to/work_with_microtvm/install_dependencies.rst
#
```

**Comment by NicolaLancellotti** [comment]
> This looks good, but do these tutorials still need to include instructions to install CMSIS? I'd love to remove this from `micro_aot.py`, `micro_autotune.py`, and `micro_tflite.py` if we can.
> 
> ```python
> ######################################################################
> #
> #     .. include:: ../../../../gallery/how_to/work_with_microtvm/install_dependencies.rst
> #
> ```

Done. Thanks for the review!

File: gallery/how_to/work_with_microtvm/install_cmsis.rst [hunk 1]
```diff
@@ -1,35 +0,0 @@
-..  Licensed to the Apache Software Foundation (ASF) under one
```
Comment (mehrdadh) [inline gallery/how_to/work_with_microtvm/install_cmsis.rst:1]: Please don't remove this file as we will need it in future tutorials

Comment (NicolaLancellotti) [reply gallery/how_to/work_with_microtvm/install_cmsis.rst]: I re-added the file and edited it adding the commands to download CMSIS NN from the new GitHub location.

**Review by mehrdadh — APPROVED** [review]

**Review by guberti — APPROVED** [review]
LGTM, merging this.

**Pull request merged**